lib/common/actions/index.js

```diff
@@ -131,6 +132,22 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    // TODO: REMOVE HACK
+    // This section (L135 - L146) is a hack to display links to patterns referenced by a stop to delete.
+    // This should be replaced with a better solution such as a new endpoint for this information.
+    if (json.detail && json.detail.includes('Referenced pattern IDs:')) {
```

Comment:
The searched string is inconsistent with the one defined at https://github.com/conveyal/gtfs-lib/blob/30754ba741ee414fc1778837160d91e12694eddf/src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java#L1616, so this will never show the links.
```suggestion
    if (json.detail && json.detail.includes('Referenced patterns:')) {
```

Reply:
Good catch the two PRs were out of sync. Updated in https://github.com/ibi-group/datatools-ui/pull/943/commits/78fc07f63988a31c5f525297b54e2db1f33d2916